kikar_hamedina/locale/he/LC_MESSAGES/django.po

```diff
@@ -169,61 +169,53 @@ msgstr "כל חברי הכנסת"
 msgid "All Members of Knesset"
 msgstr "כל חברות וחברי הכנסת"
 
-#: core/templates/core/all_members.html:12
-msgid "All the numbers in the page are referencing the monthly average."
-msgstr "כל המספרים בעמוד זה מתייחסים לממוצע החודשי."
-
-#: core/templates/core/all_members.html:13
-msgid "Grayscaled MK's don't have a public Facebook page."
-msgstr "חברות וחברי הכנסת אשר תמונתם מואפרת, לא קיים עמוד פייסבוק פומבי."
-
-#: core/templates/core/all_members.html:22
-#: core/templates/core/all_parties.html:25 core/templates/core/party.html:11
+#: core/templates/core/all_members.html:19
+#: core/templates/core/all_parties.html:26 core/templates/core/party.html:11
 msgid "Number of Facebook Pages for Party"
 msgstr "ח\"כים במפלגה"
 
-#: core/templates/core/all_members.html:25
+#: core/templates/core/all_members.html:22
 #: core/templates/core/date_filter.html:51 core/templates/core/member.html:37
 #: core/templates/core/party.html:15 core/templates/core/searchgui.html:122
 msgid "Last Month"
 msgstr "בחודש האחרון"
 
-#: core/templates/core/all_members.html:26
-#: core/templates/core/all_members.html:44
-#: core/templates/core/all_parties.html:29 core/templates/core/member.html:38
+#: core/templates/core/all_members.html:23
+#: core/templates/core/all_members.html:47
+#: core/templates/core/all_parties.html:30 core/templates/core/member.html:38
 #: core/templates/core/member.html.py:41 core/templates/core/party.html:16
 #: core/templates/core/party.html.py:19
 msgid "Statuses"
 msgstr "סטאטוסים"
 
-#: core/templates/core/all_members.html:26
+#: core/templates/core/all_members.html:23
 msgid "and"
 msgstr "וגם"
 
-#: core/templates/core/all_members.html:27
-#: core/templates/core/all_parties.html:33 core/templates/core/member.html:39
+#: core/templates/core/all_members.html:24
+#: core/templates/core/all_parties.html:34 core/templates/core/member.html:39
 #: core/templates/core/member.html.py:42 core/templates/core/party.html:17
 #: core/templates/core/party.html.py:20
 msgid "Likes on average"
 msgstr "לייקים, בממוצע"
 
-#: core/templates/core/all_members.html:47
+#: core/templates/core/all_members.html:36
+msgid "This MK doesn't have a public Facebook Page"
+msgstr "לחכ זה אין פרופיל פייסבוק פומבי"
```

Comment:
I had some issues escaping the gershayim in ח"כ - django compiled only the characters before the escaping sign.